### PR TITLE
Awesome books application.

### PR DESCRIPTION
**Pull request by aakbarkhan**
## Description:

- Add CSS styles to the application.

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @AtillaTahak & @aakbarkhan, 

**This is a great project you have going on, well-done lad**. 

   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️

- [ ] You are required to compare the feature branch to the develop branch for correct github flow. You are currently comparing the feature branch to the master branch. Make this change.
- [ ] To the readme file, since you're using the template, customize it to suit your project.
- Add a screenshot and project description.
- Add the live demo link and the video presentation if you have them, if you don't, remove these sections.
- Add a complete getting started section with all the necessary instructions and commands.

_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -6,18 +6,52 @@
...
+                  </ul>
+              </div>
+          </div>
+      </div>
+      <div class="container-fluid">
+          <div class="row justify-content-center">
+                    <h1 class="text-center text-info">ADD NEW BOOK'S</h1>
+                    <input class="form-control" type="text" name="title" id="title" placeholder="Title">
+                    <input class="form-control" type="text" name="author" id="author" placeholder="Author">
+                    <input class="btn btn-primary" type="button" value="ADD BOOK" id="add">
```
Comment (mke2111) [inline index.html]: - [x] For this button, it should be positioned in the bottom right corner just like in the given design.
Original design
Screenshot from 2021-10-28 15-37-13

Your design
Screenshot from 2021-10-28 15-37-58

File: index.html [hunk 2]
```diff
@@ -6,18 +6,52 @@
...
+          </div>
```
Comment (mke2111) [inline index.html]: - [ ] You are required to follow the general layout given.
Screenshot from 2021-10-28 15-38-54
- The book list should have equal width with the form.
- The page title should be centered.

Comment (AtillaTahak) [reply index.html]: image
No Generally layout does not have equal width the form

Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

Comment (AtillaTahak) [reply index.html]: thank you so much

File: index.html [hunk 3]
```diff
@@ -6,18 +6,59 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="css/style.css">
+
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
     <title>Awesome books</title>
 </head>
 
 <body>
```
Comment (rloterh) [inline index.html:15]: - [x] OPTIONAL
The styling of your page needs further improvement. Kindly note we actually want our design to as much as possible mimic the provided [wireframe design](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). Kindly endeavor to improve upon these aspects:
- The styling of your table **should conform to the database table alternate rows**, as can be seen in the original wireframe below. Kindly ensure to improve upon this aspect.
- The navigation, logo, and date are **not required** in this design.


**Your design:**
image

**The original wireframe design:**
image

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Atilla & Aku, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer(s). There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

_Kindly consider my comment(s) marked **[OPTIONAL]** under this review:_
<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by addod19** [comment]
### Hi @AtillaTahak, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that I'm unable to review because the repo has already been approved!!! Kindly check it out.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**